tools/glide-pindeps/glide/pin.go

```diff
@@ -0,0 +1,42 @@
+package glide
+
+import (
+	"fmt"
+	"os"
+)
+
+// PinDependencies receives a LockFile and a YamlFile and updates or inserts
+// missing imports onto the YamlFile. Dependencies with a non-empty "repo" field
+// are skipped, as they are assumed to contain openshift-specific dependencies.
+func PinDependencies(lockfile *LockFile, yamlfile *YamlFile) error {
```

Comment:
Change this signature.  I'd like it to return the `[]*YamlFileImport` it wants to add